explainer.md

```diff
@@ -687,13 +710,18 @@ enum VRFrameOfReferenceType {
 
 interface VRFrameOfReference : VRCoordinateSystem {
   readonly attribute VRStageBounds? bounds;
+  attribute EventHandler onboundschange;
 };
 
 interface VRStageBounds {
-  readonly attribute double minX;
-  readonly attribute double maxX;
-  readonly attribute double minZ;
-  readonly attribute double maxZ;
+  readonly attribute FrozenArray<VRStageBoundsPoint> geometry;
+  readonly attribute VRStageBoundsPoint playAreaMin;
+  readonly attribute VRStageBoundsPoint playAreaMax;
+};
+
+interface VRStageBoundsPoint {
+  readonly attribute double x;
+  readonly attribute double z;
```

Comment:
This may be one of those things that's a matter of personal preference... My instinct is that x,y will confuse users more than x,z ^_^